sandbox.go

```diff
@@ -263,6 +266,39 @@ func (sb *sandbox) updateGateway(ep *endpoint) error {
 	return nil
 }
 
+func (sb *sandbox) SetKey(basePath string) error {
+	var err error
+	if basePath == "" {
+		sb.Unlock()
+		return fmt.Errorf("invalid sandbox key")
+	}
+
+	sb.Lock()
+	if sb.osSbox != nil {
+		sb.Unlock()
+		return fmt.Errorf("failed to set sandbox key : already assigned")
```

Comment:
am removing the REST-API in the next update. I guess this comment is not valid after it is removed :)